server/channels/web/oauth.go

```diff
@@ -344,6 +345,42 @@ func completeOAuth(c *Context, w http.ResponseWriter, r *http.Request) {
 		} else { // For web
 			c.App.AttachSessionCookies(c.AppContext, w, r)
 		}
+
+		desktopToken := ""
+		if val, ok := props["desktop_token"]; ok {
+			desktopToken = val
+		}
+
+		if desktopToken != "" {
+			expiryTime := time.Now().Add(-model.DesktopTokenTTL).Unix()
+			desktopTokenErr := c.App.AuthenticateClientDesktopToken(desktopToken, expiryTime, user)
```

Comment:
This is not needed since we don't need to store the client desktop token on the server side at all.